### ISSUE TITLE
Unable to download cli if user used latest tag or a alpha/beta tag.

### ISSUE THREAD

**Issue by praveenkumar** [projectatomic/vagrant-service-manager#357]
**OS details:**
Linux: Fedora24

**Provider:**
VirtualBox

**Output of `vagrant -v`:**

```
N/A
```

**Output of `vagrant plugin list`:**

```
N/A
```

**Output of `vagrant service-manager box version`:**

```
Atomic Developer Bundle (ADB) 2.2.2
```

**Steps to reproduce the issue:**
1. Put **latest** tag in the Vagrantfile
2. vagrant up
3. vagrant service-manager install-cli openshift

**Describe the results you received:**

```
bad URI(is not URI?): https://github.com/openshift/origin/releases/tag/v1.3.0
.2
250
61
05
```

**Describe the results you expected:**

```
A better user warning if specific tag is not available to github.
```

**Comment by navidshaikh** [comment]
I also found same issue while testing the PR https://github.com/projectatomic/vagrant-service-manager/pull/349

**Comment by navidshaikh** [comment]
Bring up a fresh ADB box and start the openshift using VSM `start` command, test the install-cli PR and it doesn't work, since the `start` of VSM will pull the latest openshift for ADB and this PR is not working for latest (alpha, beta) version of openshift.

```
$vagrant init projectatomic/adb .

$vagrant up --provider virtualbox

$vagrant service-manager start openshift

$vagrant-service-manager git:(fix-348) ✗ bundle exec vagrant service-manager install-cli openshift
bad URI(is not URI?): https://github.com/openshift/origin/releases/tag/v1.3.0
.2
250
61
05
➜  vagrant-service-manager git:(fix-348) ✗ vagrant ssh
Last login: Mon Aug  1 05:41:29 2016 from 10.0.2.2
[vagrant@centos7-adb ~]$ oc version
oc v1.3.0-alpha.2-250-g61eba05
kubernetes v1.3.0-alpha.3-599-g2746284

[vagrant@centos7-adb ~]$ docker images
REPOSITORY                                   TAG                 IMAGE ID            CREATED             SIZE
docker.io/openshift/origin-sti-builder       latest              e3bfb30fa8d1        5 weeks ago         425.8 MB
docker.io/openshift/origin-deployer          latest              77e40fdef408        5 weeks ago         425.8 MB
docker.io/openshift/origin-docker-registry   latest              099b4288fd91        5 weeks ago         337.1 MB
docker.io/openshift/origin-haproxy-router    latest              cffa9ad573ec        5 weeks ago         445 MB
docker.io/openshift/origin                   latest              fa7bd2e9e7e1        5 weeks ago         425.8 MB
docker.io/openshift/origin-deployer          v1.3.0-alpha.2      112b6b61d786        5 weeks ago         415 MB
docker.io/openshift/origin-docker-registry   v1.3.0-alpha.2      382b5da5487b        5 weeks ago         325.7 MB
docker.io/openshift/origin-pod               v1.3.0-alpha.2      50762cad5302        5 weeks ago         1.59 MB
[vagrant@centos7-adb ~]$ 
```

**Comment by hferentschik** [comment]
Resolved via pull request #363
